test/KeyBoardActions.java

```diff
@@ -5,6 +5,9 @@
 import org.openqa.selenium.WebElement;
 import org.testng.annotations.Test;
 
+/**
+*Keyboard Actions
```

Comment:
Its Keyboard Actions of Selenium, We use the same keyboard actions for the entire Project

Reply:
Accepted Changes